src/CalcViewModel/Common/LocalizationService.h

```diff
@@ -39,6 +39,21 @@ namespace CalculatorApp { namespace Common
         Windows::UI::Text::FontWeight GetFontWeightOverride();
         double GetFontScaleFactorOverride(LanguageFontType fontType);
 
+        void Sort(std::vector<Platform::String^>& source);
+
+        template<typename T>
+        void Sort(std::vector<T>& source, std::function<Platform::String^(T)> func)
```

Comment:
Interested in making this more generic?  For example, you could accept a begin and end iterator, like STL algorithms, and then sort over that sequence.  We can probably also remove the dependency on Platform::String^, but that requires doing more work at the callsite.